validator/delphi_validator/validate.py

```diff
@@ -20,7 +20,10 @@ def __init__(self, params):
         Arguments:
             - params: dictionary of user settings; if empty, defaults will be used
         """
-        suppressed_errors =  params["global"].get('suppressed_errors', [])
+        self.export_dir = params["global"]["export_dir"]
```

Comment:
The production params files vary as to whether they store things like `export_dir` under `global` or `common`, but `common` is used more often:

```
covidcast-indicators$ grep '"common"' ansible/templates/*.j2
ansible/templates/changehc-params-prod.json.j2:  "common": {
ansible/templates/covid_act_now-params-prod.json.j2:  "common": {
ansible/templates/google_health-params-prod.json.j2:  "common": {
ansible/templates/google_health-params-test.json.j2:  "common": {
ansible/templates/google_symptoms-params-prod.json.j2:  "common": {
ansible/templates/hhs_hosp-params-prod.json.j2:  "common": {
ansible/templates/jhu-params-prod.json.j2:  "common": {
ansible/templates/nchs_mortality-params-prod.json.j2:  "common": {
ansible/templates/quidel_covidtest-params-prod.json.j2:  "common": {
ansible/templates/safegraph-params-prod.json.j2:  "common": {
ansible/templates/safegraph_patterns-params-prod.json.j2:  "common": {
ansible/templates/usafacts-params-prod.json.j2:  "common": {
covidcast-indicators$ grep '"global"' ansible/templates/*.j2
ansible/templates/usafacts-params-prod.json.j2:    "global": {
```

Reply:
There is a difference here between `common` and `validation.global` that is confusing since I think I have made some mistakes between them elsewhere.  I can rename `validation.global` to `validation.common` but it's not clear to me whether that is more or less prone to error.

Reply:
Any of the following is okay with me:
* use the same key in both locations, highlighting the similarity of purpose ("these apply to all siblings")
* use a different key in each location, highlighting the difference of purpose ("these are shared across validators; those are shared across indicator/validation/archiving")
* factor out the key name(s) into a utility function, so that it ceases to matter

...so long as we have enough test coverage to catch any errors in the params files or the code that accesses them before they reach production.

Reply:
switched to "common" everywhere